common/content/buffer.js

```diff
@@ -1648,7 +1648,7 @@ const Buffer = Module("buffer", {
 
         mappings.add(myModes, ["[["],
             "Follow the link labeled 'prev', 'previous' or '<' if it exists",
-            function (count) { buffer.followDocumentRelationship("previous"); },
+            function (count) { buffer.followDocumentRelationship("prev"); },
```

Comment:
Yes, I understand that. With this change, `[[`, instead of `<link rel="previous"`, follows `<link rel="prev"`, everything else remains the same.